src/LEGO.AsyncAPI.Bindings/Sns/OrderingConfiguration.cs

```diff
@@ -0,0 +1,20 @@
+namespace LEGO.AsyncAPI.Models.Bindings.Sns;
+
+public class OrderingConfiguration
+{
+    /// <summary>
+    /// What type of SNS Topic is this?
+    /// </summary>
+    public Ordering Type { get; set; }
+    
+    /// <summary>
+    /// True to turn on de-duplication of messages for a channel.
+    /// </summary>
+    public bool ContentBasedDeduplication { get; set; }
+}
+
+public enum Ordering
+{
```

Comment:
For enums there is a display attribute you can use. And an extension on GetScalarValue (you can take a look at the Pulsar channel binding for an example)